### ISSUE TITLE
[enhancement] a few suggestions

### ISSUE THREAD

**Issue by MartinaRuocco** [TTitcombe/docker_openai_gym#4]
Hi! 

I'm sorry if this might be a silly question (I'm new to Docker) but what do you mean by 
> Before entering the python interpreter, a script to attach the graphical display should have been run.

? Can you provide an example, please?

**Comment by TTitcombe** [comment]
No that's my fault - it's not worded very well. You don't have to do anything for that line in the documentation.

What that was saying is that it's possible to configure a dockerfile to execute something in a container before the container actually starts. The last line in the Dockerfile does this: `ENTRYPOINT ["/usr/local/bin/startup_script.sh"]`

This means that when you run the command to run the container and enter the interpreter this script will run (I said "should" because I didn't know Docker very well when I wrote this, so wasn't 100% confident). That script makes the graphical display work.

**Comment by MartinaRuocco** [comment]
Alright, thanks @TTitcombe! 

- At this point it is worth mentioning that I had to do some changes to your Dockerfile, adding "sh" as an argument to the ENTRYPOINT: 
`ENTRYPOINT ["sh","/usr/local/bin/startup_script.sh"]`
The reason is that it gave me this error `standard_init_linux.go:219: exec user process caused: no such file or directory` when launching the container (step 2 of your README). 
- I guess in step 4 and 5 you meant to write `/usr/local/bin/startup_script.sh` and not `/usr/local/bin/startup_script.py`... am I correct? 
- [testing phase] I suggest you integrate what you wrote in your (very good) [medium article](https://medium.com/@t.j.titcombe/reinforcement-learning-with-docker-789aa556504b) with the info on how to use your python script to test the container: 
  1. comment the line `shutil.rmtree(file_path)` in tests/test_image.py
  2. in a new terminal:
  `docker cp tests/test_image.py test_container:/tmp/test_image.py`
  `docker exec -it test_container /usr/local/bin/startup_script.sh "python /tmp/test_image.py"`  
  `docker cp test_container:/rec/video ~/Desktop/`

It's worth mentioning that _at the time I am writing this comment_, there is a [bug](https://stackoverflow.com/questions/65595244/record-openai-gym-video-with-monitor) in the [video_recording.py](https://github.com/openai/gym/pull/2139/commits/5c94ebabded3af1929033b72cba1c00e87c84dcf) script of the gym library that I solved by modifying it.  
For future reference, whoever wants to modify an image can follow this [guide](https://bobcares.com/blog/edit-docker-image/).
In summary:
1. run the container
`docker run -it test_container old-image-name bash`
2. From the shell, make the changes in the container as required (in this case, I modified the video_recording.py script). Once done, exit the container. 
3. from a new terminal 
`docker commit test_container new-image-name`
